src/CarGame.js

```diff
@@ -0,0 +1,43 @@
+const { isMoving, randomNumberMaker } = require("./MovementIndicator");
+
+const { COMMA } = require("./Utils/Constants");
+
+class CarGame {
+  #carStatus;
+
+  constructor() {
+    this.#carStatus = {};
+  }
```

Comment:
단순히 초기화만 하는 형태라면 생성자 함수 대신 default value를 활용하는 게 좋을 것 같습니다.

```suggestion
  #carStatus = {};
```